src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnection.cs

```diff
@@ -1000,6 +1026,33 @@ private Task WriteToStreamAsync(ReadOnlyMemory<byte> source)
             return _stream.WriteAsync(source);
         }
 
+        private bool TryReadNextLine(out ReadOnlySpan<byte> line)
+        {
+            var buffer = new ReadOnlySpan<byte>(_readBuffer, _readOffset, _readLength - _readOffset);
```

Comment:
We could use RemainingBuffer here, right?

Reply:
I guess.  I avoided doing so because I can just construct the span directly here rather than first constructing a memory and then getting its span. But it's probably inconsequential.

Reply:
Yeah, I missed the Memory/Span issue; that makes it less clear.

Neither this nor the Consume comment below are worth worrying about for this PR.  I'm more wondering how we want this to work moving forward.  But let's not worry about that for now.